### ISSUE TITLE
sort_constructors_first rule: accessors proposal

### ISSUE THREAD

**Issue by aemino** [dart-lang/sdk#57651]
After enabling the `sort_constructors_first` rule, I noticed that accessors are considered by the rule to be methods rather than properties.  I'd like to start a quick discussion about whether this should be the case.  In my code, I usually sort these first because to me, they more or less appeal as properties.  However, I can completely understand the other side of this argument.  Wondering what other people think about this.

**Comment by srawlins** [comment]
Yeah we have to leave this alone; this is inspired by a bespoke rule for a single codebase, so it's not really meant to be general purpose.
